python/test/unit/tools/test_aot.py

```diff
@@ -375,6 +376,7 @@ def test_launcher_has_no_available_kernel():
         assert "kernel launch failed" in result.stderr
 
 
+@pytest.mark.skip(reason="Slow test")
```

Comment:
FYI, this is already partially resolved here https://github.com/triton-lang/triton/pull/7068
@ThomasRaoux